docs/upgrade.html

```diff
@@ -260,17 +272,71 @@ <h5><a id="upgrade_100_new_protocols" href="#upgrade_100_new_protocols">New Prot
          be used if the SaslHandshake request version is greater than 0. </li>
 </ul>
 
-<h5><a id="upgrade_100_streams" href="#upgrade_100_streams">Upgrading a 1.0.0 Kafka Streams Application</a></h5>
+<h5><a id="upgrade_100_streams" href="#upgrade_100_streams">Upgrading a 0.11.0 Kafka Streams Application</a></h5>
```

Comment:
The title should still be `Upgrading a 1.0.0 Kafka Streams Application`?

Reply:
Actually I think the title was not consistent: previous the title of `Upgrading a <XXX> version` talks about how to upgrade from <XXX>'s previous minor release to <XXX> where here we are talking about upgrading from <XXX> to 1.0.0.

How about rename the title to `Upgrading to a 1.0.0 Kafka Streams Application`, and use a sebsuction <h6> for `Upgrading from 0.11.0`, and so on for the following sub sections.

Reply:
Yes, the "id" means upgrade to this version while the text means upgrade from this version. It's explained in the first bullet point below thought. I would prefer to do a separate PR if we want to rephrase -- this PR is already complex enough.